locales/ca/home.json

```
@@ -0,0 +1,24 @@
+{
+   "heroSection": {
+      "button": "Començar",
+      "subtitle": "Genera llistes de música a Spotify basat en les cançons que estàn sonant en aquell tour, utilizant la API de Setlist.fm",
```

```suggestion
      "subtitle": "Genera llistes de música a Spotify basades en les cançons que estan sonant en aquell tour",
```

Added a couple of minor changes.

Also: I am thinking that probably we do not need to provide details on the internals of the application,at least in the home page. Maybe we can add those details in another section.